sys/net/crosslayer/ng_pktbuf/_pktbuf_static.c

```diff
@@ -165,7 +168,7 @@ void *_pktbuf_internal_alloc(size_t size)
 
     while ((node->next != NULL)
            /* and if space between current and next allocation is not big enough */
-           && ((_start_idx(node->next) - _end_idx(node)) < _total_sz(size))) {
+           && ((_start_idx(node->next) - _end_idx(node)) < _al_sz(_total_sz(size)))) {
```

Comment:
@authmillenon I don't get why it is `start - end` (simplified). Isn't `end`always greater than `start`? And in that case you'll get negative values there? Just asking dump to get a foolproof answer :-)